packages/web3/src/contract/events.ts

```diff
@@ -1,24 +1,8 @@
-/*
```

Comment:
Q: Why do we need to change this file?

Reply:
Unresolve this issue since it is not resolved.

Reply:
It doesn't test that's why 😔